cypress/support/constants.js

```diff
@@ -19,3 +19,118 @@ export const NODE_API = {
   ...API,
   INDEX_TEMPLATE_BASE: '/_index_template',
 };
+
+export const TEST_DOCUMENT = {
+  EventTime: '2020-02-04T14:59:39.343541+00:00',
+  HostName: 'EC2AMAZ-EPO7HKA',
+  Keywords: '9223372036854775808',
+  SeverityValue: 2,
+  Severity: 'INFO',
+  EventID: 22,
+  SourceName: 'Microsoft-Windows-Sysmon',
+  ProviderGuid: '{5770385F-C22A-43E0-BF4C-06F5698FFBD9}',
+  Version: 5,
+  TaskValue: 22,
+  OpcodeValue: 0,
+  RecordNumber: 9532,
+  ExecutionProcessID: 1996,
+  ExecutionThreadID: 2616,
+  Channel: 'Microsoft-Windows-Sysmon/Operational',
+  Domain: 'NT AUTHORITY',
+  AccountName: 'SYSTEM',
+  UserID: 'S-1-5-18',
+  AccountType: 'User',
+  Message:
+    'Dns query:\r\nRuleName: \r\nUtcTime: 2020-02-04 14:59:38.349\r\nProcessGuid: {b3c285a4-3cda-5dc0-0000-001077270b00}\r\nProcessId: 1904\r\nQueryName: EC2AMAZ-EPO7HKA\r\nQueryStatus: 0\r\nQueryResults: 172.31.46.38;\r\nImage: C:\\Program Files\\nxlog\\nxlog.exe',
+  Category: 'Dns query (rule: DnsQuery)',
+  Opcode: 'Info',
+  UtcTime: '2020-02-04 14:59:38.349',
+  ProcessGuid: '{b3c285a4-3cda-5dc0-0000-001077270b00}',
+  ProcessId: '1904',
+  QueryName: 'EC2AMAZ-EPO7HKA',
+  QueryStatus: '0',
+  QueryResults: '172.31.46.38;',
+  Image: 'C:\\Program Files\\nxlog\\regsvr32.exe',
+  EventReceivedTime: '2020-02-04T14:59:40.780905+00:00',
+  SourceModuleName: 'in',
+  SourceModuleType: 'im_msvistalog',
+  CommandLine: 'eachtest',
+  Initiated: 'true',
+  Provider_Name: 'Service_ws_Control_ws_Manager',
+};
+
+export const TEST_INDEX = {
```

Comment:
Let's use types where possible

Reply:
Thus far for Cypress tests they've all been in vanilla js.  Do these tests go through tsc?

Reply:
I'm going to look for the answer to that question

Reply:
Cypress does support typescript, though we are using an older version.  There would be some set up involved to avoid errors, with things like custom commands.  For now, I need to get these tests passing on github.  Perhaps we could implement this for 2.5?

Reply:
Yeah let's have a task on us to investigate this further